src/service/data_products/heatmap_common_models.py

```diff
@@ -12,6 +13,19 @@
     description="The unique ID of the cell in the heatmap."
 )
 
+class ColumnType(str, Enum):
+    """
+    The type of a column's values.
+    """
+    FLOAT = "float",
+    """ A float. """
+    INT = "int"
+    """ An integer. """
+    COUNT = "count"
+    """ Similar to an integer, but represents a numeric count. """
```

Comment:
primary types make sense to me. But I am not sure about `count`. Do we need to build extra layers to check data quality? Because a `count` type column can have cells with float or boolean values.

Reply:
Count here is really just an int, but there's some descriptive info about the semantics of the int for the user. The microtrait type is `count` so I wanted to preserve the nomenclature for people familiar with microtrait.

How could you have a float when you're counting things? You don't count half a thing

Reply:
yea exactly, that's my question. We need to build additional checks to ensure `count` column only has integers. Currently, the cell value accepts bool, float and int.

Reply:
Oh, you're talking about error checking. For `count` I'd just make sure it's an int > 0 in the loader, which it always was for the microtrait output I looked at

Reply:
The HeatMapRow model represents what the service can send to the user, so it has to represent all possible heatmap values, but the loader knows what the types are supposed to be so the more specific check should go there

Reply:
👍 got it.